previews/primer/alpha/tooltip_preview.rb

```diff
@@ -7,7 +7,7 @@ class TooltipPreview < ViewComponent::Preview
       # @param type [Symbol] select [["Description", description], ["Label", label]]
       # @param direction select [s, n, e, w, ne, nw, se, sw]
       # @param tooltip_text text
-      def playground(type: :description, direction: :s, tooltip_text: "Tooltip text")
+      def playground(type: :description, direction: :s, tooltip_text: "You can press a button")
```

Comment:
Why are all these copy changes necessary?

Reply:
I forget what the rule name was but it's essentially that the screenreader text needs to include the text of the element so that there isn't a huge mismatch in the information conveyed to users.